### PR TITLE
Addressed some comments from PR #625

### PR DESCRIPTION
**Pull request by ravil-mobile**

### CODE REVIEW

**Review by Thomas-Ulrich — APPROVED** [review]

File: src/DynamicRupture/Output/ReceiverBasedOutput.cpp [hunk 1]
```diff
@@ -142,87 +146,89 @@ void ReceiverBasedOutput::calcFaultOutput(const OutputType type,
...
+                                                      rotatedInitStress[QuantityIndices::XX];
     }
 
-    auto& ruptureVelocity = std::get<VariableID::RuptureVelocity>(outputData.vars);
+    auto& ruptureVelocity = std::get<VariableID::RuptureVelocity>(outputData->vars);
     if (ruptureVelocity.isActive) {
-      auto& jacobiT2d = outputData.jacobianT2d[i];
-      ruptureVelocity(level, i) = this->computeRuptureVelocity(jacobiT2d);
+      auto& jacobiT2d = outputData->jacobianT2d[i];
+      ruptureVelocity(level, i) = this->computeRuptureVelocity(jacobiT2d, local);
```
[the remark this answers is not in the document]
Comment (ravil-mobile) [reply src/DynamicRupture/Output/ReceiverBasedOutput.cpp:216]: Hi.

In this implementation the firsts arguments are read-write values following to the read-only values.

> What motivates local being not any more a member of the class?

File: src/DynamicRupture/Output/ReceiverBasedOutput.cpp [hunk 2]
```diff
@@ -142,87 +146,89 @@ void ReceiverBasedOutput::calcFaultOutput(const OutputType type,
 
     switch (generalParams.slipRateOutputType) {
     case SlipRateOutputType::TractionsAndFailure: {
-      this->computeSlipRate(rotatedUpdatedStress, rotatedStress);
+      this->computeLocalSlipRate(local, rotatedUpdatedStress, rotatedStress);
```
[the remark this answers is not in the document]
Comment (ravil-mobile) [reply src/DynamicRupture/Output/ReceiverBasedOutput.cpp:149]: I added local to underline that I am updating `local` variable.

[the remark this answers is not in the document]
Comment (Thomas-Ulrich) [reply src/DynamicRupture/Output/ReceiverBasedOutput.cpp:149]: then maybe should be updateLocalSlipRate? or computeSlipRateAndUpdateLocal

File: src/DynamicRupture/Output/ReceiverBasedOutput.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (Thomas-Ulrich) [reply src/DynamicRupture/Output/ReceiverBasedOutput.cpp:216]: I see, thank you.

**Pull request merged**